src/azlinter.ts

```diff
@@ -0,0 +1,25 @@
+import { Host } from '@azure-tools/autorest-extension-base';
+import { AzConfiguration, CodeGenConstants, PathConstants } from './utils/models';
+import * as path from 'path';
+import { runLintball } from './utils/helper';
+
+export async function processRequest(host: Host) {
+    try {
+        const folder = await host.GetValue(CodeGenConstants.azOutputFolder);
+        const azextFolder = AzConfiguration.getValue(CodeGenConstants.azextFolder);
+        let fileName = path.join(
```

Comment:
can we put this in the 'else' ?

Reply:
I found a way to combine the two file path logic in different modes